### PR TITLE
Schemas.update

### PR DESCRIPTION
**Pull request by flavio-fernandes**
- add check for GOPATH
- update schemas

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,9 @@ SCHEMA?=schemas/ovn-nb.ovsschema
 all: build
 
 $(MODEL_GEN):
+ifeq ($(GOPATH),)
+	  $(error GOPATH is not set)
```
Comment (flavio-fernandes) [inline Makefile:13]: NIT: Let me fix this spacing.

Comment (flavio-fernandes) [reply Makefile]: done

**Comment by amorenoz** [comment]
Please rebase on top of recently merged #15 . Hopefully that will fix the ci issue

**Comment by amorenoz** [comment]
LGTM
Thanks @flavio-fernandes !!

**Pull request merged**